src/components/navigation/NavigationKeyHandler.ts

```diff
@@ -41,24 +60,24 @@ export function levelNav(
 			document.activeElement?.classList.contains("a11y_row") ||
 			document.activeElement?.id === "guide_close"
 		) {
-			return;
+			return number;
 		}
 		if (document.activeElement?.classList.contains("a11y_desc") && alertDiv) {
 			alertDiv.textContent = "You can only change series while focused on a data point";
 			setTimeout(() => {
```

Comment:
Checkout the [`wait` function on the `@feedzai/js-utilities` package.](https://feedzai.github.io/js-utilities/docs/functions/DOM/wait)
It is a promise that wraps around a timeout, but makes your code look cleaner.

Reply:
I have identified this and other places in the code where it makes sense to use js-utilities functions. I believe it would be best to submit a separate PR for these changes in the future.